src/error.rs

```diff
@@ -0,0 +1,49 @@
+use goblin::error::Error as GoblinError;
+use pdb_crate::Error as PDBError;
+pub type CompactSymbolResult<T> = Result<T, CompactSymbolError>;
```

Comment:
You can call this `Result`:

```rust
type Result<T> = std::result::Result<T, GetSymbolsError>;
```

Reply:
With that type declaration, we wouldn't be able to abstract `Result` to other future classes of errors.